src/index.html

```diff
@@ -2,11 +2,153 @@
 <html>
   <head>
     <meta charset="UTF-8">
+    <link rel="stylesheet" href="../styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Ubuntu&display=swap" rel="stylesheet">
     <title>Whats Cookin</title>
   </head>
   <body>
-    <h1>Whats Cookin</h1>
 
```

Comment:
I like having the spaces between the sections in HTML while we are working. We just have to remember to remove them before our final push when the project is due!